packages/bls-did-provider-key/src/BlsKeyDidResolver.ts

```diff
@@ -0,0 +1,24 @@
+import { resolve } from '@transmute/did-key.js'
```

Comment:
What is the need or this whole module whilst Veramo already provides https://veramo.io/docs/api/did-provider-key.getdidkeyresolver/ ? Which is also described in our main readme: https://github.com/Sphereon-Opensource/ssi-sdk#did-resolution

Reply:
Veramo's implementation does not resolve dids created with a bls keys. I had to add '@transmute/did-key-bls12381' and 'did:key:zUC7': resolveBls12381G2 to the startWithMap

Reply:
Okay. That would probably make a good PR upstream in the future.